AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -258,6 +262,22 @@ - (void)signOut {
 
 #pragma mark - Private methods
 
+- (BOOL)checkURLSchemeRegistered:(NSString *)urlScheme {
+  NSArray *schemes;
+  NSString *typeRole;
+  NSArray *types = [MS_APP_MAIN_BUNDLE objectForInfoDictionaryKey:kMSCFBundleURLTypes];
+  for (NSDictionary *urlType in types) {
+    schemes = urlType[kMSCFBundleURLSchemes];
```

Comment:
I think this will crash if `urlType` is not an `NSDictionary`, and same if `scheme` and `typeRole` are not `NSString`

Reply:
Or if `scheme` is not an `NSArray`. Basically, should we type check these?

Reply:
`CFBundleURLTypes` is an array of dictionaries. [Apple Doc](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleurltypes)
`CFBundleURLSchemes` is an array of strings. [Apple Doc](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleurltypes/cfbundleurlschemes)